src/components/FormInput.tsx

```diff
@@ -1,29 +1,48 @@
 import React, { useState } from 'react';
+import TodoListProps from '../TypescriptProps';
 
 interface FormInputProp {
     addTask: any;
```

Comment:
to specify a function type here use 
```
addTask: (userInput: string ) => void;
```
This specifies a function type which takes a string as a parameter and returns nothing
again avoid any

Reply:
I updated this function type to string. Resolving out.

Reply:
I was referring to a different review, this one is not yet resolved. Still encountering errors.